pkg/cmd/server/kubernetes/master.go

```diff
@@ -97,6 +105,23 @@ func (c *MasterConfig) RunPersistentVolumeClaimRecycler(recyclerImageName string
 	recycler.Run()
 }
 
+// attemptToLoadRecycler tries decoding a pod from a filepath for use as a recycler for a volume.
+// If a path is not set as a CLI flag, no load will be attempted and no error returned.
+// If a path is set and the pod was successfully loaded, the recycler pod will be set on the config and no error returned.
+// Any failed attempt to load the recycler pod will return an error.
+func attemptToLoadRecycler(path string, config *volume.VolumeConfig) error {
```

Comment:
I had considered that and agree it would be better factored that way.  It would require another carried patch, though, so I thought a future PR to tidy this up after it lands during a normal Kube rebase.